app/models/imported_client.rb

```diff
@@ -4,17 +4,12 @@ class ImportedClient < NonHmisClient
       data_source_id: DataSource.where(db_identifier: ImportedClient.ds_identifier).select(:id),
     )
   end, foreign_key: :id_in_data_source, required: false
+  has_many :client_assessments, class_name: 'ImportedClientAssessment', foreign_key: :non_hmis_client_id, dependent: :destroy
+  has_one :current_assessment, -> { order(created_at: :desc) }, class_name: 'ImportedClientAssessment', foreign_key: :non_hmis_client_id
```

Comment:
I _think_ we've determined ordering by id is generally faster and returns the same result.